flexget/utils/bittorrent.py

```diff
@@ -223,15 +223,23 @@ def get_filelist(self):
         files = []
         if 'length' in self.content['info']:
             # single file torrent
-            t = {'name': self.content['info']['name'],
+            if 'name.utf-8' in self.content['info']:
+                name = self.content['info']['name.utf-8']
+            else:
+                name = self.content['info']['name']
+            t = {'name': name,
```

Comment:
I think I prefer `'name': self.content['info'].get('name.utf-8') or self.content['info']['name']` or `'name': self.content['info'].get('name.utf-8', self.content['info']['name'])`, but that's purely a stylistic preference.

Reply:
My python is limited 😄 it does indeed look cleaner